ui/inspectors/service.reel/dyndns-service.reel/dyndns-service.js

```diff
@@ -0,0 +1,18 @@
+var Component = require("montage/ui/component").Component,
+    ServiceDyndnsProvider = require("core/model/enumerations/service-dyndns-provider").ServiceDyndnsProvider;
+
+/**
+ * @class DynamicDnsService
+ * @extends Component
+ */
+exports.DynamicDnsService = Component.specialize({
+    providerOptions: {
+        value: null
+    },
+    
+    templateDidLoad: {
+        value: function() {
+          this.providerOptions = ServiceDyndnsProvider.members;
```

Comment:
The middleware enum contains `null`, it should be wrapped to display a better label to the user.
Something like
```
this.providerOptions = ServiceDyndnsProvider.members.map(function(x) { 
    return { 
        value: x, 
        label: x || 'None' 
    }
});
```
might do the job.